packages/web-components/src/global/_variables.scss

```diff
@@ -138,7 +138,7 @@
     */
 
     // --color-border-menu: #2b659b;
-    // --color-menu-item-hover: #1c3f5e4d;
+    --color-menu-item-hover: #1c3f5e4d;
     // --color-segmented-button-selected: var(--color-snowflakes-dark-selected);
```

Comment:
This is should probably belong as an alias in the theme files. It's also one of the cases where we need to decide if we are going to apply opacity separate from a color value or include it. This is --color-snowflakes-dark-selected with an opacity of 0.3. No need to change anything now but we should remember this example for the design token discussion.

Reply:
Does this fall into one of our defined colors? ie `--colorPrimaryLighten1` or something similar? Or is it a snowflake?

Reply:
it's `--selected`, youll want to just swap 

` --popup-menu-item-hover-background-color: var(--color-menu-item-hover);` for ` --popup-menu-item-hover-background-color: var(--selected);`

--color-menu-item-hover can be removed, it was a temp var until pop up was done in figma

Reply:
`--selected` doesn't have an opacity tho

Reply:
my b, try --list-hover

Reply:
Sweet, thanks. Now go enjoy your vacation!